internal/db/schema/migrations/postgres/11/01_server_type_enum.up.sql

```diff
@@ -0,0 +1,17 @@
+begin;
+
+create table server_type_enm (
+  name text primary key not null,
+  constraint only_predefined_server_types_allowed
+    check (name = ANY (array['controller'::text, 'worker'::text]))
```

Comment:
why not use the current pattern and just do `name in (...)`?  Is there some advantage to using `ANY` and an array?

Reply:
Hmm... this might be just how psql writes this and not from a pattern I saw. Will fix!

Reply:
Updated!